Source/iop/Iop_Spu2.h

```diff
@@ -32,6 +32,16 @@ namespace Iop
 			REGS_END = 0x1F90FFFF,
 		};
 
+		enum REGISTERS
+		{
+			// SPDIF interface
+			SPDIF_OUT = 0x1F9007C0, // SPDIF Out: OFF/'PCM'/Bitstream/Bypass

```

Comment:
Can we move the definitions for these in the `enum` that already has `C_IRQINFO`?

Can we also rename the definitions to match these? https://github.com/ps2dev/ps2sdk/blob/60fa652b3ec082f6871e7d00af06ce7b1b3eeaa3/iop/sound/libsd/include/spu2regs.h#L85

I know `SPDIF_PROTECT` isn't there, but we can name that one `C_SPDIF_PROTECT`.